crates/egui/src/containers/window.rs

```diff
@@ -356,6 +356,13 @@ impl<'open> Window<'open> {
         self
     }
 
+    /// If there is [`ScrollArea`], For each axis, should the containing area shrink if the content is small?
+    #[inline]
+    pub fn auto_shrink(mut self, auto_shrink: impl Into<Vec2b>) -> Self {
+        self.scroll = self.scroll.auto_shrink(auto_shrink);
+        self
+    }
+
```

Comment:
```suggestion
```

Reply:
The new code you suggested is the same as the original one.  There seems to be no changes in the suggestion.